pkg/controller/certificates/issuing/issuing_controller_test.go

```diff
@@ -290,6 +290,59 @@ func TestIssuingController(t *testing.T) {
 								ObservedGeneration: 3,
 							}),
 							gen.SetCertificateLastFailureTime(metaFixedClockStart),
+							gen.SetCertificateIssuanceAttempts(pointerToInt(1)),
```

Comment:
https://github.com/kubernetes/utils/tree/master/pointer
```suggestion
							gen.SetCertificateIssuanceAttempts(pointer.Int(1)),
```

Reply:
This is nice